AssertionError: Test case is leaking resources

Apparently occurs when a resource has not been consumed or closed (see denoland/deno#4735).
_await conn.close()_ did not work.

**Code:**

```
Deno.test("Connection to unavailable db", async () => {
  const conn = new Database("mongo", {
    uri: "mongodb://localhost:27017",
    database: "unavailable-db",
  });

  const ping = await conn.ping();

  await conn.close();

  assertEquals(
    ping,
    false,
    "Database unavailable-db exists or connection to an unavailable db successful",
  );
});
```

**Error:**

AssertionError: Test case is leaking resources.
Before: {
  "0": "stdin",
  "1": "stdout",
  "2": "stderr"
}
After: {
  "0": "stdin",
  "1": "stdout",
  "2": "stderr",
  "3": "tcpStream"
}

Make sure to close all open resource handles returned from Deno APIs before
finishing test case.
    at assert (deno:runtime/rt/06_util.js:33:13)
    at Object.resourceSanitizer [as fn] (deno:runtime/rt/40_testing.js:81:7)
    at async TestRunner.[Symbol.asyncIterator] (deno:runtime/rt/40_testing.js:245:13)
    at async Object.runTests (deno:runtime/rt/40_testing.js:322:22)
    at async file:///mnt/e/Personal%20Projects/deno/denodb/denodb-playground/$deno$test.ts:3:1

**deno --version**
deno 1.6.1 (release, x86_64-unknown-linux-gnu)
v8 8.8.278.2
typescript 4.1.2

--- a/lib/connectors/mongodb-connector.ts
+++ b/lib/connectors/mongodb-connector.ts
@@ -289,6 +289,7 @@ export class MongoDBConnector implements Connector {
       return;
     }
 
+    this._client.close();
     this._connected = false;
   }
 }

